lib/ui/views/messenger/layouts/create_discussion_validation_sheet.dart

```diff
@@ -0,0 +1,207 @@
+                children: [
+                  BackButton(
+                    key: const Key('back'),
+                    color: theme.text,
+                    onPressed: () {
+                      Navigator.pop(context);
+                    },
+                  ),
+                  AutoSizeText(
+                    localizations.newDiscussion,
+                    style: theme.textStyleSize24W700EquinoxPrimary,
+                    textAlign: TextAlign.center,
+                    maxLines: 1,
+                    stepGranularity: 0.1,
+                  ),
+                ],
+              ),
+              Visibility(
+                visible: formState.membersList.length > 1,
+                child: AppTextField(
+                  leftMargin: 0,
+                  rightMargin: 0,
+                  labelText: localizations.name,
+                  onChanged: (text) {
+                    formNotifier.setName(text);
+                  },
+                  controller: nameController,
+                ),
+              ),
+              const SizedBox(
+                height: 30,
+              ),
+              Text(
+                localizations.aboutToCreateADiscussion,
+                style: theme.textStyleSize14W600Primary,
+              ),
+              const SizedBox(
+                height: 15,
+              ),
+              Expanded(
+                child: ListView.builder(
+                  itemCount: formState.membersList.length,
+                  itemBuilder: (context, index) {
+                    return Row(
+                      children: [
+                        Text(
+                          formState.membersList[index].format,
+                          style: theme.textStyleSize16W700Primary,
+                        ),
+                        const Expanded(child: SizedBox()),
+                        IconButton(
+                          onPressed: () {
+                            Sheets.showAppHeightNineSheet(
+                              context: context,
+                              ref: ref,
+                              widget: ContactDetail(
+                                contact: formState.membersList[index],
+                                editMode: false,
+                              ),
+                            );
+                          },
+                          icon: const Icon(Icons.info_outline),
+                        ),
+                      ],
+                    );
+                  },
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(top: 15),
+                child: Align(
+                  alignment: Alignment.bottomCenter,
+                  child: Row(
+                    children: <Widget>[
+                      AppButtonTiny(
+                        AppButtonTinyType.primary,
+                        localizations.createDiscussion,
+                        const [0, 0, 0, 0],
```

Comment:
Could you create in `Dimens` class a constant about the margin with 0, 0, 0, 0 like `  static const List<double> none = <double>[0, 0, 0, 0];`